home/templates/course.html

```diff
@@ -6,7 +6,42 @@
 {% block description %}View professor reviews and grade data for {{ course.name }} at the University of Maryland &mdash; College Park{% endblock %}
 
 {% block head %}
+
 <script type="text/javascript" src="{% static 'js/chart.min.js' %}"></script>
+<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```

Comment:
I can see `base_main.html` already contains 
`<script type="text/javascript" src="{% static 'js/jquery-3.5.1.min.js' %}"></script>`

is there any particular reason we want to introduce `https://code.jquery.com/jquery-3.6.0.min.js` here?